iOS/G3MiOSSDK/Commons/Rendererers/BusyMeshRenderer.cpp

```diff
@@ -71,9 +71,9 @@ Mesh* BusyMeshRenderer::createMesh(const G3MRenderContext* rc) {
   const int viewPortWidth  = camera->getViewPortWidth();
   const int viewPortHeight = camera->getViewPortHeight();
   const int minSize = (viewPortWidth < viewPortHeight) ? viewPortWidth : viewPortHeight;
-  const float outerRadius = minSize / 15.0f;
-
-  const IMathUtils* mu = IMathUtils::instance();
+  const float outerRadius = minSize / 18.0f;
+    ILogger::instance()->logError("This a test log line");
```

Comment:
Hi @DrakkLord 

Can you, please, roll it back to the values in purgatory?   After that, I think we can merge this PR.